src/components/Header/index.tsx

```diff
@@ -239,7 +239,7 @@ export default function Header() {
         </Title>
         <NavigationLinks />
         <HeaderControls>
-          <HeaderElement>
+          {/* <HeaderElement>
```

Comment:
Delete this element instead of commenting it. Do not leave unnecessary comments in the codebase, unless the comment is needed to provide more context or description of the method / code.